kubernetes/kustomize/cornserve/base/resource-manager/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       nodeSelector:
         node-role.kubernetes.io/control-plane: "true"
-      serviceAccountName: resource-manager
+      serviceAccountName: cornserve-sa
```

Comment:
Restore service-specific service accounts (e.g., `resource-manager`) and instead create a new `RoleBinding` object that gives CRD access to `resource-manager` service account.